target/linux/ramips/image/mt7621.mk

```diff
@@ -699,6 +699,16 @@ define Device/gnubee_gb-pc2
 endef
 TARGET_DEVICES += gnubee_gb-pc2
 
+define Device/hanyang_hyc-g920
+  $(Device/dsa-migration)
+  DEVICE_VENDOR := Hanyang
+  DEVICE_MODEL := CJ-Hello HYC-G920
```

Comment:
Umm, in the DW02-412H support patch, we did use `DEVICE_VENDOR` for the real manufacturer ("Hanyang" in this case) and `DEVICE_ALT0_*` for the ISP ("CJ-Hello"). I'm still not sure if this is the best approach, but I don't have a better idea.
https://github.com/openwrt/openwrt/blob/e3e9eb31a281643737142e8e80c6f49204b5ba18/target/linux/ath79/image/nand.mk#L94-L97

Reply:
> Umm, in the DW02-412H support patch, we did use `DEVICE_VENDOR` for the real manufacturer ("Hanyang" in this case) and `DEVICE_ALT0_*` for the ISP ("CJ-Hello"). I'm still not sure if this is the best approach, but I don't have a better idea.
> 
> https://github.com/openwrt/openwrt/blob/e3e9eb31a281643737142e8e80c6f49204b5ba18/target/linux/ath79/image/nand.mk#L94-L97

Device Model is HYC-G920 
if you want to exclude the ISP name it is OK 
and for KT-412 there 2 ISPs 
Olleh GiGA WiFi home 
and 
KT GiGA WiFi home

for this Device, ISP name Printed on top of Device ( CJ-Hello ) and on the Back Model is HYC-G920
with FCC ID MSIP-CMM-HYD-HYC-G920

Reply:
If the ISP did not add another model name ("CJ-Hello Hello" sounds weird...), then I would not add `DEVICE_ALT0_*` for it. (No ISP name)